solutions/standard/main.tf

```diff
@@ -232,3 +232,83 @@ data "ibm_database_connection" "existing_connection" {
   user_id       = data.ibm_database.existing_db_instance[0].adminuser
   user_type     = "database"
 }
+
+########################################################################################################################
+# Code Engine Kibana Dashboard instance
+########################################################################################################################
+
+data "http" "es_metadata" {
+  count    = var.enable_kibana_dashboard ? 1 : 0
+  url      = "https://${local.es_username}:${local.es_password}@${local.es_host}:${local.es_port}"
+  insecure = true
```

Comment:
Did you face some issue when insecure was set to false?

Reply:
Yes, it throws an errror. If [insecure](https://registry.terraform.io/providers/hashicorp/http/latest/docs/data-sources/http#insecure) is `false` that means it is going to verify server's certificate chain and hostname which we are not setting in DA.

```
 2024/10/21 14:48:41 Terraform apply | Error making request: GET
 2024/10/21 14:48:41 Terraform apply | https://admin:xxxxx@5d899ad6-472c-42e1-8603-616a061a96a4.bn2a2uid0up8mv7mv2ig.private.databases.appdomain.cloud:31243
 2024/10/21 14:48:41 Terraform apply | giving up after 1 attempt(s): Get
 2024/10/21 14:48:41 Terraform apply | "https://admin:***@5d899ad6-472c-42e1-8603-616a061a96a4.bn2a2uid0up8mv7mv2ig.private.databases.appdomain.cloud:31243":
 2024/10/21 14:48:41 Terraform apply | tls: failed to verify certificate: x509: certificate signed by unknown
 2024/10/21 14:48:41 Terraform apply | authority
```

Reply:
And can we not pull the cert and do a secure connection? client cert is generated as part of service credential creation as far as I know

Reply:
I need to check how we can download the TLS cert file and then pass it to http data source.

Reply:
Done, testing.